File: python_programming/practices/financial_calculator.py
# KR 6th Financial calculator

income = round(float(input("What is your monthly income?:")), 2)
rent = round(float(input("What is your monthly rent/mortgage?:")),2)
utilities = round(float(input("What is your monthly utilities?:")), 2)
groceries = round(float(input("What is you rmonthly groceries?:")), 2)
transport = round(float(input("What is your monthly transport?:")), 2)
savings = int(income*0.1)

rent_perc = int(rent/income*100)
util_perc = int(utilities/income*100)
groc_perc = int(groceries/income*100)
tran_perc = int(transport/income*100)
save_perc = int(savings/income*100)

print("your rent is", rent, "and that is", rent_perc, "percent of your income")
print("your utilities is", utilities, "and that is", util_perc, "percent of your income")
print("your groceries is", groceries, "and that is", groc_perc, "percent of your income")
print("your tansport is", transport, "and that is", tran_perc, "percent of your income")
print("you should save", savings, "a month, and that is", save_perc, "percent of your income")

total = rent+utilities+groceries+transport+savings
s_m = round(float(income-total), 2)

print("you have", s_m, "dollars of spending money each month")

File: c_programming/practices/financial_calculator.c
//KR 6th Financial Calculator
#include <stdio.h>

int main(void){
    int income;
    int rent;
    int utilities;
    int groceries;
    int transport; 

    printf("What is your income?\n");
    scanf("%d", &income);
    //while(getfloat() != '\n')
    printf("What is your monthly rent?\n");
    scanf("%d", &rent);
    printf("What is your monthly utilities cost?\n");
    scanf("%d", &utilities);
    printf("What is your monthly groceries cost?\n");
    scanf("%d", &groceries);
    printf("What is your monthly transport cost?\n");
    scanf("%d", &transport);

    int savings = income*0.1;

    float rent_percent = (float)rent/income*100;
    float utilities_percent = (float)utilities/income*100;
    float groceries_percent = (float)groceries/income*100;
    float transport_percent = (float)transport/income*100;
    float savings_percent = (float)savings/income*100;


    printf("your rent is %d and that is %1.0f percent of your income.\n", rent, rent_percent);
    printf("your utilites is %d and that is %1.0f percent 30of your income.\n", utilities, utilities_percent);
    printf("your groceries is %d and that is %1.0f percent of your income.\n", groceries, groceries_percent);
    printf("your transport is %d and that is %1.0f percent of your income.\n", transport, transport_percent);
    printf("you should save %d a month and that is %1.0f percent of your income.\n", savings, savings_percent);

    int total = savings+transport+groceries+utilities+rent;
    int spending_money = income-total;

    printf("you have %d of spending money each month!", spending_money);

    return 0;
}